### DISCUSSION TITLE
No "-config" option in the standard command "ecparam". - OpenSSL 3.2.2 4

### DISCUSSION THREAD

**Discussion by Snugi** [openssl/openssl#26292] [Q&A] [answered]
Hello OpenSSL-Team.

I mine, I have to found a Bug in the use case with the parameter "ecparam" and the option "-config".

It's easy to say something is a bug when I don't know if it's intended by development or not.

**Here the Facts**
OpenSSL Version: OpenSSL 3.2.2 4 Jun 2024 (Library: OpenSSL 3.2.2 4 Jun 2024)

**I specify the Problem**
If I use the "ecparam" Standard command with the Option "-config /etc/myRootCA.conf" I became a error. The config can't be read. The rights are 744. I read the help for "openssl ecparam" with the command "openssl ecparam -help". There is no option to use an existing configuration file. Is  it intended?

In the following I use a scheme the first Line is the commando and in the second line it is the error message.

**Command:**
`sudo openssl ecparam -genkey -noout -out /home/CA/CAs/GuettingerCA/private/01gRootCA.key -config /home/CA/CAs/01gRootCA.conf`
**Error:**
`ecparam: Unknown option: -config`

Is there a way to use a configuration file with "ecparam"?
How long it goes to programming this option into a new version of OpenSSL?

Kind regards
Snugi

**Comment by t8m** [comment, marked as answer]
This is not a bug. There `ecparam` command does not have a `-config` option. What do you want to configure in the config file for the ecparam command?

You can change the location of the default config file by setting the environment variable OPENSSL_CONF.
